### ISSUE TITLE
Fail build when assetCompile fails

### ISSUE THREAD

**Issue by erichelgeson** [bertramdev/asset-pipeline#196]
asset-pipeline should fail the build if `:assetCompile` has an error.

For example I accidentally used a ES6 feature in ES5 mode. It produced an empty file but did not fail the build so it went unnoticed.

```
notifications.unminified.js:2: ERROR - this language feature is only supported for ECMASCRIPT6 mode or better: const declaration. Use --language_in=ECMASCRIPT6 or ECMASCRIPT6_STRICT to enable ES6 features.
    const timeoutMs = 30000;
    ^
1 error(s), 0 warning(s)
```

Grails 3.3.x
AP 2.14.6

**Comment by makfair** [comment]
Hi, 
There was a similar issue reported here which was closed, from the comment on close looks like not a proper response. Anything can be done to resolve this issue?
https://github.com/bertramdev/asset-pipeline/issues/101

**Comment by davydotcom** [comment]
Seems to be related to the ExecutorService not capturing the Exceptions… looking into it

**Comment by davydotcom** [comment]
Fixed in soon to be released 3.0.2

**Comment by pelenthium** [comment]
Hi, does anyone know about this issue? 3.0.6 don't break the build if assetCompile have errors
